.github/workflows/doc.yml

```diff
@@ -53,6 +53,28 @@ jobs:
     - name: 'Build HTML documentation'
       run: make -C Doc/ SPHINXOPTS="-q" SPHINXERRORHANDLING="-W --keep-going" html
 
+    # Add pull request annotations for Sphinx nitpicks (missing references)
+    - name: Get list of changed files
```

Comment:
```suggestion
    - name: 'Get list of changed files'
```

If we're going to run with the single-quotes convention for names used in the rest of the file, and in keeping with this PR being "nit-picky" :), may as well be consistent here.

Reply:
Why do changes for pure consistency that adds zero benefit?

Reply:
Because it wouldn't be a change; this entry is newly added with this PR, and the `name`s in the existing file (and the other `name`s added in this PR) are all quoted. I wouldn't have suggested it otherwise, and even so I hadn't bothered until Ezio brought it up above.

Reply:
My IDE (PyCharm) highlights explicit strings in green instead of white, so that helps pick out them out, I guess.